runner/status.go

```diff
@@ -26,12 +26,12 @@ const (
 	ABORTED
 	// Operation timed out and was killed.
 	TIMEDOUT
-	// Invalid or error'd request. Original runner state not affected. Retry may work after mutation.
+	// Request rejected for reasons unrelated to the user.
```

Comment:
I think we should remove "un/related to the user," as that just causes confusion.  BADREQUEST is a specific type of failure that prevented receipt of a request (it looks like we use this if we can't convert from thrift to domain object, which makes sense).  I don't have a problem with the categories as-is but the comment was misleading.